### PR TITLE
Update create animal form #166

### PR DESCRIPTION
**Pull request by linaskitm**
Something wrong with description. 

Screenshot 2021-05-19 165401

### CODE REVIEW

**Comment by saulyz** [comment]
Regarding your question about the screenshot - is the problem the `description` value? Normally would expect `string` with HTML and it seems we have `object`. This might depend upon the component configuration. Can't give the ready solution, but exploring https://github.com/niuware/mui-rte might help.

**Comment by domzic** [comment]
mui-rte editor passes stringified object to `onSave` callback. We need to convert it to JSON object and then take `text` field.
Could you add these changes?
`onSave={data => props.onChange(JSON.parse(data).text)}`

File: src/components/animal/create/DetailsStep.tsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ function DetailsStep({ onNext }: DetailsStepProps) {
                     />
                 </Grid>
                 <Grid item xs={12} sm={6}>
-                    <TextInput name="imageUrl" label="Image URL" id="imageUrl" fullWidth />
+                    <TextInput type="file" name="imageUrl" id="imageUrl" fullWidth />
```
Comment (domzic) [inline src/components/animal/create/DetailsStep.tsx:95]: Do not set `type="file"`. We wan't just image URL, not to upload raw image

**Review by domzic — CHANGES_REQUESTED** [review]

File: src/components/animal/create/DetailsStep.tsx (hunk 1 above)
Comment (saulyz) [reply src/components/animal/create/DetailsStep.tsx:95]: @domzic sorry, there should be `type="file"` according to the last specification from the backend. So this change is actually valid. @linaskitm but the name then is not good - should not have "Url" as this is the file now.

Comment (linaskitm) [reply src/components/animal/create/DetailsStep.tsx]: Thanks for comments, I changed the image name.  
And  `onSave={data => props.onChange(JSON.parse(data).text)}` doesn't work as expected.

**Review by domzic — APPROVED** [review]

**Comment by saulyz** [comment]
Merging this PR as it has reached main objectives.
mui-rte fixes will be made on another PR.

**Pull request merged**